experimental/examples/logs/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "logs-example",
```

Comment:
If the example is made to be typescript, it can be added to the lerna project and built with other examples to ensure it stays up to date.

Reply:
Previously, I referred to other example codes that were implemented in JavaScript.  I agree with your opinion, and I have modified them to be implemented in TypeScript.